docker/compose/validator-testnet/docker-compose.yaml

```diff
@@ -1,49 +1,80 @@
-# This compose file defines a single validator node test network. In addition, it offers the
```

Comment:
might recommend we have a single node testnet docker and a more complete ecosystem.

Reply:
have gone back and forth on that with @xli- general consensus so far has been generally if you're launching a testnet you want a faucet: in the event you don't, you can always ignore it or change the `docker-compose` command. In the spirit of keeping things simple, I would advocate for keeping them together as is and potentially creating additional `docker-compose` if we have the specific need. How does that sound?

Reply:
Hmm... I guess if we run this all in the background and still export the root key, it is fine either way.